server/controllers/crm/crm4Controller.ts

```diff
@@ -16,12 +16,32 @@ export default class Crm4Controller {
       const crm4Response = await this.crm4Service.getCrm(usn)
       const navigation = this.crmDisplayService.getCrmNavigation('crm4', usn, sectionId, crm4Response)
       const section = this.crmDisplayService.getCrmSection('crm4', sectionId, crm4Response)
+
+      const history = req.session.history || []
```

Comment:
This code is repeated in crm5Controller, suggest moving to a function to be called from both crm4Controller & crm5Controller.

Reply:
done